src/pages/GoalDetail/components/TodoBox/index.tsx

```diff
@@ -1,36 +1,88 @@
 import { Todo } from '@/types/interface';
 import { PlusIcon } from '@assets';
 import TodoItem from '@components/TodoItem';
+import TodoCreateModal from '@components/TodoModal/TodoCreateModal';
+import { AxiosHeaders, AxiosRequestConfig } from 'axios';
+import { useState } from 'react';
+import InfiniteScroll from 'react-infinite-scroller';
 
+type PageData = {
+  nextCursor: number | null;
+  todos: Todo[];
+  totalCount: number;
+};
+
+type Page = {
+  config: AxiosRequestConfig;
+  data: PageData;
+  headers: AxiosHeaders;
+  request: XMLHttpRequest;
+  status: number;
+  statusText: string;
+};
 interface TodoBoxProps {
   title: 'To do' | 'Done';
   placeholder: string;
-  todos: Todo[];
+  todos: any;
```

Comment:
[import question] 이거  rule 안끄셨는데 warning이라서 안잡힌걸까요? ? 상단에 룰 끄시는게 나을거 같아요 아니면 이렇게 쓰면 안될까요?? 
```suggestion
  todos: {
    pages: Page[];
  };
```

Reply:
타입이 상당히 복잡해서 저렇게는 안되고 일단 룰 꺼두겠슴다